### ISSUE TITLE
[Simple Commander] TB3 isn't rendered correctly in Gazebo

### ISSUE THREAD

**Issue by ljaniec** [ros-planning/navigation2#2699]
## Bug report
- operating system: Ubuntu 20.04
- ROS2 Galactic (binaries)

#### Steps to reproduce:
- install `aws-robomaker-small-warehouse-world` and `nav2-simple-commander` ROS2 Galactic packages (binaries)
- `ros2 launch nav2_simple_commander inspection_demo_launch.py` (BTW the command in README is outdated, I think? with `ros2 launch nav2_simple_commander demo_security_launch.py` - it isn't working)

Part of my `.bashrc` file:
```
source ~/ros2_ws/install/setup.bash
export TURTLEBOT3_MODEL=waffle
export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:/home/ljaniec/ros2_ws/install/aws_robomaker_small_warehouse_world/share/aws_robomaker_small_warehouse_world/worlds:/home/ljaniec/ros2_ws/install/aws_robomaker_small_warehouse_world/share/aws_robomaker_small_warehouse_world/models:/home/ljaniec/ros2_ws/install/neo_simulation2/share/neo_simulation2/robots:/home/ljaniec/ros2_ws/install/neo_simulation2/share/neo_simulation2/models:/opt/ros/galactic/share/turtlebot3_gazebo/models:/home/ljaniec/.gazebo/models
```

#### Expected behavior

The simulation with one visible TB3 rendered and Nav2 stack + demo script start.

#### Actual behavior

The simulation doesn't have correctly rendered robot (screens below). Autonomy demo still works.
-

![Simulation 1](https://user-images.githubusercontent.com/11331846/142863407-5728a14c-7299-48fe-b092-4d123ea749d1.png)

![Simulation 2](https://user-images.githubusercontent.com/11331846/142863429-424e2a5a-9590-4453-88aa-8f840be9035a.png)

![Simulation 3](https://user-images.githubusercontent.com/11331846/142863445-a441cf08-a093-415d-b875-505d839755f4.png)

![Simulation 4](https://user-images.githubusercontent.com/11331846/142863459-cc1f8213-3779-4135-9619-47b899197de2.png)

#### Additional information

What I tried:

- reinstall Gazebo and Nav2 packages

- I have found a similar bug [there](https://github.com/mkrizmancic/franka_gazebo/issues/1) 

- I checked URDF like [there](http://gazebosim.org/tutorials/?tut=ros_urdf) and it didn't show problems

- this bug shows in the original package too (`nav2_rosdevday_2021`, [there](https://github.com/SteveMacenski/nav2_rosdevday_2021/issues/7) ), one with MP 400 (the robot is invisible, in "Collisions" view in Gazebo too - unlike there with TB3) - maybe it is something connected with these meshfile changes mentioned in the first link above?

Am I doing something wrong there? Standard simulation (1 TB3 in TurtleWorld) is working fine - robot model loads etc.

**Comment by ljaniec** [comment]
Hello,

I prepared a small set of commands to show, what I think the problem is.

**TLDR;** URDF models of TB3 in `nav2_bringup` and of MP400 from [Neobotix](https://github.com/neobotix/neo_simulation2)

## TB3 case 1

### Start Gazebo simulation with empty world

`ros2 launch gazebo_ros gazebo.launch.py`

### Add TB3 SDF model to Gazebo (package turtlebot3_gazebo)

`ros2 run gazebo_ros spawn_entity.py -file /opt/ros/galactic/share/turtlebot3_gazebo/models/turtlebot3_waffle/model.sdf -entity robot01`

It spawns TB3 Waffle normally (visible model and LiDAR).

## TB3 case 2-1

### Start Gazebo simulation with empty world

`ros2 launch gazebo_ros gazebo.launch.py`

### Add TB3 URDF model to Gazebo (package nav2_bringup)

`ros2 run gazebo_ros spawn_entity.py -file /opt/ros/galactic/share/nav2_bringup/urdf/turtlebot3_waffle.urdf -entity robot01`

Surprisingly, it spawns invisible TB3 Waffle. It is visible only in the 'Collision' view in Gazebo. No LiDAR too.

## TB3 case 2-2

### Start Gazebo simulation with empty world

`ros2 launch gazebo_ros gazebo.launch.py`

### Generate TB3 SDF model from URDF

`gz sdf -p turtlebot3_waffle.urdf > turtlebot3_waffle.sdf`

### Add TB3 SDF model to Gazebo (from package nav2_bringup)

`ros2 run gazebo_ros spawn_entity.py -file ~/Desktop/turtlebot3_waffle.sdf -entity robot01`

TB3 is invisible (but it's there, after launching 'Collision' view in Gazebo).

## MP400 case

### Start Gazebo simulation with empty world

`ros2 launch gazebo_ros gazebo.launch.py`

### Add MP400 URDF model to Gazebo

`ros2 run gazebo_ros spawn_entity.py -file ~/mp_400/mp_400.urdf -entity robot01`

Robot is invisible, there are only blobs in the air and a LiDAR.

### Generate SDF model from URDF

`gz sdf -p mp_400.urdf > mp_400.sdf`

### Add MP400 SDF model to Gazebo

`ros2 run gazebo_ros spawn_entity.py -file ~/mp_400/mp_400.sdf -entity robot01`

Robot is again invisible, there are only blobs in the air and a LiDAR.

I think this is still relevant to mention it there, because this TB3's URDF model is used in a few places in the Nav2 demos. 

Relevant issues and links I have found:
* https://github.com/ros-planning/navigation2/issues/2157 - but it's is solved - there is `/opt/ros/galactic/share/turtlebot3_gazebo` already
* https://answers.gazebosim.org//question/26073/cannot-spawn-urdf-into-gazebo-using-ros2/ and https://answers.gazebosim.org//question/2282/convert-urdf-to-sdf-or-load-urdf/ - these two 
* https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1284 - it looks like this could be the problem addressed it this (unmerged yet) PR ("[gazebo_ros_factory]: Resolve package:// urls")

**Comment by ljaniec** [comment]
Relevant issue: https://github.com/ros-simulation/gazebo_ros_pkgs/issues/1272#issuecomment-857513366
